o/opentelemetry-go/build_info.json

```diff
@@ -2,14 +2,13 @@
     "package_name" : "opentelemetry-go",
     "github_url":"https://github.com/open-telemetry/opentelemetry-go",
     "version": "v1.11.1",
-    "package_dir": "o/opentelemetry-go/latest",
+    "package_dir": "o/opentelemetry-go",
     "docker_build": false,
     "build_script" : "opentelemetry-go_ubi_8.5.sh",
-    
+
     "v1.*.*" : {
- 
-	"args": {
-	    "GO_VERSION":"1.18.5"
+            "args": {
+            "GO_VERSION":"1.18.5"
```

Comment:
@Pratikt2312 we will need to support older version also. If you keep version pattern `v1.*` only , it will not support older version `v0.13.0, v0.20.0` which are supported in current build scripts. Please handle both cases. 
You can keep two different entries as there is a change in GO version also.

Reply:
Yes I checked it for older versions like v0.13.0,v0.20.0,etc.and also made the required changes in json file.